packages/firebase_admob/android/build.gradle

```diff
@@ -32,6 +32,6 @@ android {
         disable 'InvalidPackage'
     }
     dependencies {
-        api 'com.google.firebase:firebase-ads:15.+'
+        api 'com.google.firebase:firebase-ads:15.0.0'
```

Comment:
Note that the version of this dependency has been bumped to 15.0.1, rebasing on master should resolve this.

Reply:
Used latest sdk version.